src/Tribe/Autoloader.php

```diff
@@ -163,11 +163,25 @@ private function normalize_root_dir( $root_dir ) {
 
 			protected function get_prefixed_path( $class ) {
 				foreach ( $this->prefixes as $prefix => $dirs ) {
+					$is_namespaced = false !== strpos( $prefix, '\\' );
+
+					if ( $is_namespaced ) {
+						// If the prefix is a namespace, then normalize it.
+						$prefix = trim( $prefix, '\\' ) . '\\';
```

Comment:
On further review, this would be useful if `$this->prefixes` already had the prefix normalized by this point as it's referenced via `get_prefix_by_slug()` elsewhere and the returned prefix would need normalizing there manually for each use.

Reply:
Abstracted this in https://github.com/moderntribe/tribe-common/pull/1350 so we can remove this if conditional that normalizes here.